src/vmm/src/lib.rs

```diff
@@ -162,14 +162,13 @@ pub type EventManager = BaseEventManager<Arc<Mutex<dyn MutEventSubscriber>>>;
 // clippy lint `upper_case_acronyms` we have disabled this lint for this enum.
 /// Vmm exit-code type.
 #[allow(clippy::upper_case_acronyms)]
-#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+#[derive(Debug, Clone, Copy, PartialEq, Eq, thiserror::Error, displaydoc::Display)]
```

Comment:
This one is also not really an error type - it contains both an "Ok" as well as "Error" variants, so it play the role of a "Result" instead (we are using it as an Error type in some places, but that's tech debt we're trying to get rid of, so let's avoid deriving `Error` here)

Reply:
Updated this change as well!